lib/Dist/Zilla/Plugin/Alien.pm

```diff
@@ -330,6 +357,18 @@ sub _helper_hash {
 	\%helper;
 }
 
+has env => (
+	isa => 'ArrayRef[Str]',
+	is => 'rw',
+	default => sub { [] },
+);
+
+sub _env_hash {
+	my($self) = @_;
+	my %env = map { /^\s*(.*?)\s*=\s*(.*)\s*$/ ? ($1 => $2) : ($_ => '') } @{ $self->env };
```

Comment:
Yeah I am aware of that.  Having a default value was just for consistency with the other hash like configuration thingies.  I don't think there really is another sensical default value.  Maybe we don't allow an environment variable without a specified value?